tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/volume_utilization_handler.py

```diff
@@ -63,7 +63,7 @@ def format_alert(self, alert_json):
                     "Volume utilization on %s in "
                     "%s back to normal" % (
                         alert['tags']['volume_name'],
-                        alert['tags']['integration_id']
+                        alert['tags']['cluster_short_name']
```

Comment:
why are you assuming "cluster_short_name" will always be set? 

what happens at this line if there is no "cluster_short_name"

Reply:
If cluster_short_name is not present then I will assign  integration_id in alert['tags']['cluster_short_name']